x.toml

```diff
@@ -156,3 +156,26 @@ members = [
     "secure/key-manager",
     "storage/backup/backup-cli",
 ]
+
+# ---
+# Determinator rules
+# ---
+
+# CI-related files. TODO: maybe have separate rules for local and CI?
+[[determinator.path-rule]]
+globs = [".github/**/*", ".circleci/**/*", "codecov.yml"]
+mark-changed = "all"
+
+# Core Rust and devtools files.
+[[determinator.path-rule]]
+globs = ["rust-toolchain", "cargo-toolchain", "cargo-flags", "x.toml"]
+mark-changed = "all"
+
+[[determinator.path-rule]]
+# Ignore website changes in x.
+globs = ["developers.libra.org/**/*"]
+mark-changed = []
+
+[[determinator.package-rule]]
+on-affected = ["x"]
+mark-changed = "all"
```

Comment:
Is x considered a package of libra?    I'd think the virtual workspace would ignore it for the purposes of say "cargo build" or "cargo test".   I think we'd also want to remove x as da default member? https://github.com/libra/libra/blob/master/Cargo.toml#L45-L47.

Reply:
> Is x considered a package of libra?

Yes, it's in the same workspace, just like all of our other non-test code.

> I'd think the virtual workspace would ignore it for the purposes of say "cargo build" or "cargo test".

Well, we do want tests in `x` to run in CI...

> I think we'd also want to remove x as default member?

Don't have a strong opinion either way on this at the moment -- it's kind of a production binary in that it is a core part of the production pipeline.